files/mods.json

```diff
@@ -61,7 +61,7 @@
         ],
         "display": "GUI Bundle",
         "enabled": true,
-        "description": "Useful GUI related features, e.g a custom main menu, a custom chat GUI, custom report and reward claim GUI and more.",
+        "description": "Some GUI-related tweaks, like improving the chat experience",
```

Comment:
chat part is great sure but I feel like the custom and improved (eg smooth scrolling) menus might be nice to mention as well